src/main/java/com/alkemy/ong/service/impl/SlideServiceImpl.java

```diff
@@ -67,7 +68,33 @@ public Long saveSlide(SlideDto dto) {
 		return slidesDetailDto;
 	}
 
+	@Transactional
 	@Override
+	public void updateSlides(SlideDto dto, Long id) {
+		
+		repository.findById(id).map(slide -> {
+			slide.setImageUrl(amazonService.uploadImage64(dto.getImageB64()));
+			slide.setOrder(dto.getOrder());
+			slide.setOrganization(orgService.findById(dto.getOrganizationId()));
+			slide.setText(dto.getText());
+			
+			if (dto.getOrder() == null) {
+				Integer maxOrder = repository.findMaxOrder(dto.getOrganizationId());
+				if (maxOrder == null) {
+					slide.setOrder(1);
+				} else {
+					slide.setOrder(maxOrder + 1);
+				}
+			}
+			
+			repository.save(slide);
```

Comment:
You may be missing a return here. It should be:
`return repository.save(slide);`